cmd/test_coverage/coverage.go

```diff
@@ -265,60 +265,114 @@ func coverage(c *cli.Context) error {
 		}
-		for _, test := range append(vagrantCoverage, intCoverage...) {
-			var testGroup string
-			if group := "integration"; strings.Contains(test.shortPath, group) {
-				testGroup = group
-			} else if group := "install"; strings.Contains(test.shortPath, group) {
-				testGroup = group
-			} else if group := "e2e"; strings.Contains(test.shortPath, group) {
-				testGroup = group
+		fmt.Printf("Unused flags:\n\n")
+		for _, f := range unusedFlags {
+			fmt.Println(f)
+		}
+	}
+
+	if c.Bool("table") {
+		fmt.Printf("\n| Server flags: | | | \n| - | - | - |\n")
+		for i := 0; i < len(usedFlags); i += 3 {
+			checkBoxHtml0 := fmt.Sprintf("<ul><li>- [x] %s</li></ul>", usedFlags[i])
+			if i+2 < len(usedFlags) {
+				checkBoxHtml1 := fmt.Sprintf("<ul><li>- [x] %s</li></ul>", usedFlags[i+1])
+				checkBoxHtml2 := fmt.Sprintf("<ul><li>- [x] %s</li></ul>", usedFlags[i+2])
+				fmt.Printf("| %s | %s | %s |\n", checkBoxHtml0, checkBoxHtml1, checkBoxHtml2)
+			} else if i+1 < len(usedFlags) {
+				checkBoxHtml1 := fmt.Sprintf("<ul><li>- [x] %s</li></ul>", usedFlags[i+1])
+				fmt.Printf("| %s | %s | |\n", checkBoxHtml0, checkBoxHtml1)
+			} else {
+				fmt.Printf("| %s | | |\n", checkBoxHtml0)
 			}
+		}
 
-			flagHits := make([]int, len(xFlagNames))
-			for i, flag := range xFlagNames {
-				if test.serverArguments[flag] {
-					flagHits[i] = 1
-				}
+		for i := 0; i < len(unusedFlags); i += 3 {
+			checkBoxHtml0 := fmt.Sprintf("<ul><li>- [ ] %s</li></ul>", unusedFlags[i])
+			if i+2 < len(unusedFlags) {
+				checkBoxHtml1 := fmt.Sprintf("<ul><li>- [ ] %s</li></ul>", unusedFlags[i+1])
+				checkBoxHtml2 := fmt.Sprintf("<ul><li>- [ ] %s</li></ul>", unusedFlags[i+2])
+				fmt.Printf("| %s | %s | %s |\n", checkBoxHtml0, checkBoxHtml1, checkBoxHtml2)
+			} else if i+1 < len(unusedFlags) {
+				checkBoxHtml1 := fmt.Sprintf("<ul><li>- [ ] %s</li></ul>", unusedFlags[i+1])
+				fmt.Printf("| %s | %s | |\n", checkBoxHtml0, checkBoxHtml1)
+			} else {
+				fmt.Printf("| %s | | |\n", checkBoxHtml0)
 			}
-			data = append(data, &grob.Bar{
-				Name:        test.shortPath,
-				X:           xFlagNames,
-				Y:           flagHits,
-				Type:        grob.TraceTypeBar,
-				Legendgroup: testGroup,
-			})
 		}
+	}
 
-		fig := &grob.Fig{
-			Data: data,
-			Layout: &grob.Layout{
-				Title: &grob.LayoutTitle{
-					Text: "Server Argument Coverage",
-				},
-				Xaxis: &grob.LayoutXaxis{
-					Tickangle: 60,
-				},
-				Yaxis: &grob.LayoutYaxis{
-					Title: &grob.LayoutYaxisTitle{
-						Text: "# of Tests Using Flag",
-					},
-				},
-				Barmode: "stack",
-			},
+	return nil
+}
+
+func graphResults(serverFlagSet map[string]int, vagrantCoverage []TestCov, intCoverage []TestCov) {
+	data := grob.Traces{}
+	xFlagNames := []string{}
+
+	for k := range serverFlagSet {
+		xFlagNames = append(xFlagNames, k)
+	}
+	for _, test := range append(vagrantCoverage, intCoverage...) {
+		var testGroup string
+		if group := "integration"; strings.Contains(test.shortPath, group) {
```

Comment:
Another suggestion, this code couldn't be simplified ? Example:

`var testGroup string
if strings.Contains(test.shortPath, "integration") {
    testGroup = "integration"
} else if strings.Contains(test.shortPath, "install") {
    testGroup = "install"
} else if strings.Contains(test.shortPath, "e2e") {
    testGroup = "e2e"
}`

Reply:
Good call